freshmaker/producer.py

```diff
@@ -48,7 +47,8 @@ class FreshmakerProducer(PollingProducer):
 
     def poll(self):
         try:
-            self.check_unfinished_koji_tasks(db.session)
+            with app.app_context():
```

Comment:
If this is required, it probably means we have to check all db session commits to make sure they're called with proper app context.

Reply:
You are absolutely right!

Just for the record, we talked and decided to refrain from updating the flask and sqlalchemy packages, as that would require a thorough testing of all db operations (since the next version is a major-udpate). I reverted this change and created a new task in Jira for updating the db packages in the future, if we see fit.